### ISSUE TITLE
Rydde opp etter Okctl

### ISSUE THREAD

**Issue by yngvark** [oslokommune/kjoremiljo-admin-issues#97]
# Beskrivelse

- Ta ned dokumentasjon (ghost.io)
- gI BE

# Motivasjon

Vi vil ikke ha gammelt skrot liggende.

**Comment by yngvark** [comment]
Sjekk med Ximena om det er noe compliance greier vi må vite om

**Comment by leiacf** [comment]
Har snakket såvidt med Ximena, men vi må ha et møte. Litt tett kalender så det tar litt tid, men det viktigste er: 

**ikke slett noe dokumentasjon før vi vet om vi må arkivere**..

**Comment by deifyed** [comment]
metrics er tilgjengelig her: https://oslokommune.slack.com/archives/C018EFCSC4W/p1675344430057789

**Comment by deifyed** [comment]
> Gi beskjed til teamene om at de må fjerne Okctl (ta ned clustre, fjerne secrets, etc)

føler at den er litt detaljestyrete.. mistenker at teamene har god nok hygienekultur at de kommer til å klare det fint uten mas

**Comment by yngvark** [comment]
Yeah. Mulig de ikke vet om de ressursene som ligger igjen da, som secrets og evt. andre unmanaged ressurser (det var noe mer rest tror jeg).

**Comment by deifyed** [comment]
PDFer av nedlastet okctl.io finnes her: https://drive.google.com/drive/folders/1XZpSmW36s1csIH-XDd-D-p49bZiSlClH?usp=share_link

**Comment by leiacf** [comment]
Bør vi rydde i slack-kanalene og aws-kontonavn?

**Comment by yngvark** [comment]
Lagde #108 for AWS-kontonavn.

**Comment by leiacf** [comment]
Flytter denne ut av "now" fordi jeg har litt for mange ting på blokka som må komme før jeg kan få snakket med Ximena. Tar tak i det igjen når det roer seg litt.

**Comment by leiacf** [comment]
Mangler bare å snakke med Ximena igjen om arkiveringen. Jeg har satt det opp som agendapunkt på neste 1-1
